tests/api/commerce_decoupled_checkout/DonationCest.php

```diff
@@ -118,14 +135,10 @@ public function donationSingleExampleSuccess(\ApiTester $I) {
     $post['order']['field_appeal'] = $this->appeal->id();
     $I->sendPOST('/commerce/order/create', $post);
 
-    /** @var \Drupal\commerce_store\StoreStorageInterface $commerce_store */
-    //$commerce_store = \Drupal::entityTypeManager()
-    //  ->getStorage('commerce_store');
-
-    //Debug::debug($commerce_store->loadDefault());
-
     $I->expectTo('See successful response.');
     $I->seeResponseCodeIs(HttpCode::CREATED);
+
+    $this->orderId =$I->grabDataFromResponseByJsonPath("$.order_id[0].value")[0];
```

Comment:
Missing space after `=`.
Just curious why you didn't use "$.order_id[0].value[0]" instead of "$.order_id[0].value".

Reply:
Because in response it looks like "order_id":[{"value":3}]. 
when i use  "$.order_id[0].value" i get looks like [0=3].
when i use  "$.order_id[0].value[0]" i get nothing.
It happened because method grabDataFromResponseByJsonPath return array of found entities. In my case we found 1 entity, so we get first entity from array.

Reply:
Gotcha! Thanks for explanation @PavelBulat!